web/src/SecureVote/Eth/Models.elm

```diff
@@ -6,16 +6,19 @@ type alias CandidateEthTx =
     , value : Int
     , to : Maybe String
     , data : Maybe String
+    , gasLimit : Maybe String
```

Comment:
this should be `gas` not `gasLimit` https://github.com/ethereum/wiki/wiki/JavaScript-API#contract-methods

Reply:
Also, just set to a string - it's constant